week_2_workflow_orchestration/README.md

```diff
@@ -76,10 +76,25 @@
 
 :movie_camera: [Video](https://www.youtube.com/watch?v=gGC23ZK7lr8&list=PL3MmuxUbc_hJed7dXYoJw8DoCuVHhGEQb&index=22)
 
+* [Prefect docs](https://docs.prefect.io/)
+* [Pefect Discourse](https://discourse.prefect.io/)
+* [Prefect Cloud](https://app.prefect.cloud/)
+* [Prefect Slack](https://prefect-community.slack.com)
 
 ### Homework 
 
-TBA
+To be posted on Monday by 14:00 CET
+
+
+### Extra Practice
+
+1. Using Prefect flows and tasks with the *green* instead of *yellow* taxi dataset, ingest the data into your data lake in Google Cloud Storage, transform it, and put put it into BigQuery. 
```

Comment:
I think this would be a good homework

Reply:
Like all these steps - let's put them to homework and come up with questions that are easy to grade based on each of these actions